### ISSUE TITLE
TextGrid error on loading SantaBarbara corpus

### ISSUE THREAD

**Issue by james-tanner** [MontrealCorpusTools/SPADE#11]
Python output:
```
Processing...
Resetting the corpus.
loading
loading /projects/spade/repo/git/spade-SantaBarbara/ with <polyglotdb.io.parsers.mfa.MfaParser object at 0x7f5a6f36fe48>
Finding  files...
Parsing types...
Parsing types from file 1 of 60...
('/projects/spade/repo/git/spade-SantaBarbara/audio_and_transcripts', 'sbc021.TextGrid')
Traceback (most recent call last):
  File "duration.py", line 89, in <module>
    common.loading(config, corpus_conf['corpus_directory'], corpus_conf['input_format'])
  File "/home/linguistics/jtanner/SPADE/Common/common.py", line 126, in loading
    c.load(parser, corpus_dir)
  File "/home/linguistics/jtanner/PolyglotDB/polyglotdb/corpus/importable.py", line 132, in load
    could_not_parse = self.load_directory(parser, path)
  File "/home/linguistics/jtanner/PolyglotDB/polyglotdb/corpus/importable.py", line 212, in load_directory
    information = parser.parse_information(path, self.corpus_name)
  File "/home/linguistics/jtanner/PolyglotDB/polyglotdb/io/parsers/base.py", line 209, in parse_information
    data = self.parse_discourse(path, types_only=True)
  File "/home/linguistics/jtanner/PolyglotDB/polyglotdb/io/parsers/aligner.py", line 93, in parse_discourse
    raise (TextGridError('This file ({}) cannot be parsed by the {} parser.'.format(path, self.name)))
polyglotdb.exceptions.TextGridError: This file (/projects/spade/repo/git/spade-SantaBarbara/audio_and_transcripts/sbc021.TextGrid) cannot be parsed by the MFA parser.
```

YAML file:
```
corpus_directory: "/projects/spade/repo/git/spade-SantaBarbara/"
input_format: "MFA"
dialect_code: "gam"
unisyn_spade_directory: "/home/linguistics/jtanner/SPADE/"
speaker_enrichment_file: "/projects/spade/repo/git/spade-SantaBarbara/corpus-data/enrichment/speaker_info.csv"
speakers: []
vowel_inventory: ["ER0", "IH2", "EH1", "AE0", "UH1", "AY2", "AW2", "UW1", "OY2", "OY1", "AO0", "AH2", "ER1", "AW1",
                   "OW0", "IY1", "IY2", "UW0", "AA1", "EY0", "AE1", "AA0", "OW1", "AW0", "AO1", "AO2", "IH0", "ER2",
                   "UW2", "IY0", "AE2", "AH0", "AH1", "UH2", "EH2", "UH0", "EY1", "AY0", "AY1", "EH0", "EY2", "AA2",
                   "OW2", "IH1"]
stressed_vowels: ["EH1", "UH1", "UW1", "OY1", "ER1", "AW1", "IY1", "AA1", "AE1", "OW1",  "AO1", "AH1", "EY1", "AY1", "IH1"]
extra_syllabic_segments: []
sibilant_segments: ["S", "Z", "SH", "ZH"]
pauses: '^<SIL>$'
```

**Comment by james-tanner** [comment]
I think the reason why `formant.py` works for @mmcauliffe is because it's being called from the `/media/share/corpora/` directory instead of `/projects/spade/`. When I try `.formant.py` with the other directory I run into permissions errors as it's calling files from inside `data/mmcauliffe/polyglot_data/`.

**Comment by mmcauliffe** [comment]
The Santa Barbara corpus in the spade repo directory is not the aligned version, it's the pre aligned version.  Update the TextGrids and it will work.

**Comment by james-tanner** [comment]
Moved TextGrids over from `/media/share/corpora/SantaBarbara_aligned/`, but import fails as there are (what seem to be) issues with the TextGrids themselves. In `sbc034.TextGrid`, for example, the import returns
```    
raise ValueError(self.maxTime)
ValueError: 1480.44503
```
which matches the same error listed in `/media/share/corpora/SantaBarbara_aligned/output_errors.txt` for that TextGrid. I've also tried opening the TextGrid in Praat, which also returns an error and can't be opened. That suggests that these TextGrids are themselves malformed (and not simply erroneous parsing).

@msonderegger Since there are only 3 malformed TextGrids (according to `output_errors.txt`), would it make sense to simply skip these during export and exclude them?
